### ISSUE TITLE
Report on budget spend

### ISSUE THREAD

**Issue by rbankston** [rbankston/budget-buddy-project#6]
Indicate to the user how much money has been spent

- Determine an easy-to-understand layout for this report.
- User should be able to see how much money is being spent on each of the four categories.

**Comment by rbankston** [comment]
| Entertainment| Food | Clothing | Bills |
|----|------|-----|-----|
| $ | $ | $ | $|

**Comment by rbankston** [comment]
![image](https://user-images.githubusercontent.com/130836/182980764-063ae21b-9f8e-4704-b495-403e9bd2ea15.png)

**Comment by Jacob-Coffey** [comment]
I have the code to get the total spent on each individual category as well as the total spent. With this is one extra line to calculate the remaining available balance based on the input weekly budget of our user. I also set up a basic form with inputs to test my code.

***The problem I am currently running in to, is having the selected expense type be logged. So if I select Bills as the category, I am not able to get my code to understand that I selected Bills.
I am using radio buttons to allow the user to select between the four categories***
